appserver/website-bootstrap/network.html

```diff
@@ -46,6 +47,156 @@ <h3>Analyze Network</h3>
             <div class="col-12 col-md-10">
                 <header class="page-header header container-fluid">
                     <svg id="network"></svg>
+					<script>
+
+						var width = 900, height = 600;
+
+						var svg = d3.select("#network")
+							.attr("width", width)
+							.attr("height", height);
+
+						var simulation = d3.forceSimulation()
+							.force("link", d3.forceLink().id(function(d) { return d.id; }))
+							.force("charge", d3.forceManyBody()
+								.strength(-300)
+								.theta(0.8)
+								.distanceMax(150)
+							)
+							.force("center", d3.forceCenter(width / 2, height / 2));
+
+						const graph = {
+							"nodes": [
+								{"id": "1"},
+								{"id": "2"},
+								{"id": "3"},
+								{"id": "4"},
+								{"id": "5"},
+								{"id": "6"},
+								{"id": "7"},
+								{"id": "8"},
+								{"id": "9"},
+								{"id": "10"},
+								{"id": "11"},
+								{"id": "12"},
+								{"id": "13"},
+								{"id": "14"},
+								{"id": "15"},
+								{"id": "16"},
+								{"id": "17"},
+								{"id": "18"},
+								{"id": "19"},
+								{"id": "20"}
```

Comment:
Would it be much trouble to rename these ids to something like "n#" (e.g. "n1", "n2", "n3", etc.)? I'm concerned that having ids that were simply numbers would have an elevated risk of causing confusion.

Reply:
you got it